### PR TITLE
Menu Update

### PR DESCRIPTION
**Pull request by KarolDenst**
Added Controls section.
Now the application will launch without the game loaded if no argumant is provided.
Added reloading the menu after something is changed

### CODE REVIEW

File: Common/GeometryType.cs
Comment (AleksyBalazinski) [inline Common/GeometryType.cs:1]: Can we rename to `SelectedGeometryType`? I think `None` makes more sense in this context

File: Hyper/Window.cs [hunk 1]
```diff
@@ -96,18 +110,17 @@ protected override void OnKeyDown(KeyboardKeyEventArgs e)
...
             CursorState = CursorState == CursorState.Grabbed ? CursorState.Normal : CursorState.Grabbed;
             return;
         }
 
-        if (_game.IsRunning)
+        if (_isGameRunning)
         {
-            _game.KeyDown(e.Key);
+            _game?.KeyDown(e.Key);
             if (e.Key == Keys.T)
```
Comment (AleksyBalazinski) [inline Hyper/Window.cs:121]: Is it used?

File: Hyper/Window.cs [hunk 2]
```diff
@@ -96,18 +110,17 @@ protected override void OnKeyDown(KeyboardKeyEventArgs e)
 
         if (e.Key == Keys.Escape)
```
Comment (AleksyBalazinski) [inline Hyper/Window.cs:111]: this behaves weird when starting the game, probably should return early if GeometryType==None or something

File: Hyper/CommandInterpreter.cs
[the remark this answers is not in the document]
Comment (KarolDenst) [reply Hyper/CommandInterpreter.cs:1]: I think we can change the requirements and I don't see anyone using it if menu does all the things CLI does

File: Hud/Widgets/Colors/ColorGetter.cs [hunk 3]
```diff
@@ -8,9 +8,9 @@ public static class ColorGetter
 {
     public static Vector4 GetVector(Color color) => color switch
```
Comment (AleksyBalazinski) [inline Hud/Widgets/Colors/ColorGetter.cs:9]: can we please also change the green and red to something more eye-hurting? like
        Color.Red => new Vector4(1, 0.2f, 0.2f, 1),
        Color.Green => new Vector4(0.58f, 0.85f, 0.46f, 1),

**Pull request merged**